### DISCUSSION TITLE
In the project homepage, there are some difference between the code file structure and download file structure

### DISCUSSION THREAD

**Discussion by flyly0755** [sqlalchemy/sqlalchemy#7870] [Usage Questions] [answered]
<img width="1171" alt="image" src="https://user-images.githubusercontent.com/45039060/160624685-463a0aec-b3fa-4192-8ba2-70f1009cd95e.png">

I was just a little confused with  this screenshot showed as above,
why the code file structure in the homepage is different with the real downloaded file structure,
you can see database this dir cannot be seen in the page, and the same is processors.py this file.
At the same time, the dir name "cyextension", i think is a spelling mistake with an extra char 'y'. 
this is't any issue with the code, but by accident found these discrepancy, so post here and let you guys know it, maybe some issue with github:)

**Comment by zzzeek** [comment, marked as answer]
cyextension is a new package that replaces cextensions.  it's cython code.

processors.py is removed from the top level directory so this would be an artifact in your repo, use "git clean" to remove it.

**Reply by zzzeek** [reply]
that is, after checking out the main branch from git.  your screenshot looks like the rel_1_4 branch.

**Reply by CaselIT** [reply]
it actually seems like a merge of 1_4 and main, since cyextension does not exist in 1_4.

@flyly0755 I would suggest removing the current folder and cloning / downloading it again

**Reply by flyly0755** [reply]
ok, thank you all
